src/client/Health/NodeView.jsx

```diff
@@ -79,7 +79,7 @@ const NodeView = ({ status, onRestart, onShutdown }) => {
         label: (
           <div className={`oi-box tight text-${dataSource.enabled ? 'success' : 'muted'}`}>
             <Link to={`/south/${dataSource.dataSourceId}`}>
-              <div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>{dataSource.dataSourceId}</div>
+              <div id="overlay">{dataSource.dataSourceId}</div>
```

Comment:
Why do you use an id ? Id should be use uniquely, but here you will have several id in your code with the same reference. What you need is a class, not an id